solidity/contracts/bridge/BridgeState.sol

```diff
@@ -175,11 +175,20 @@ library BridgeState {
         // Value in seconds.
         uint32 walletCreationPeriod;
         // The minimum BTC threshold in satoshi that is used to decide about
-        // wallet creation or closing.
-        uint64 walletMinBtcBalance;
+        // wallet creation. Specifically, we allow for the creation of a new
+        // wallet if the active wallet is old enough and their amount of BTC
+        // is greater than or equal this threshold.
+        uint64 walletCreationMinBtcBalance;
         // The maximum BTC threshold in satoshi that is used to decide about
-        // wallet creation.
-        uint64 walletMaxBtcBalance;
+        // wallet creation. Specifically, we allow for the creation of a new
+        // wallet if the active wallet's amount of BTC is greater than or equal
+        // this threshold, regardless of the active wallet's age.
+        uint64 walletCreationMaxBtcBalance;
```

Comment:
RFC 1 tells about `wallet_min_creation_btc` and `wallet_min_closure_btc` but nothing about `wallet_max_creation_btc`. As far as I recall, it was removed some time ago. We need to decide if it's needed or not and aligned the RFC and implementation cc @beaushinkle

Reply:
IIRC it was not mentioned in the RFC but decided out of band to have this parameter in order to allow creating new wallets regardless of the active wallet age once the active wallet has too much BTC.